pgsql/expected/pointcloud.out

```diff
@@ -362,8 +362,8 @@ DELETE FROM pa_test_dim;
 INSERT INTO pa_test_dim (pa)
 SELECT PC_Patch(PC_MakePoint(3, ARRAY[x,y,z,intensity]))
 FROM (
- SELECT  
- -127+a/100.0 AS x, 
+ SELECT
+ -127+a/100.0 AS x,
```

Comment:
Do you mind removing this change? It's unrelated, and it should be fixed upstream.